pom.xml

```diff
@@ -574,11 +586,27 @@
         <artifactId>scala-guice_${scala.version.prefix}</artifactId>
         <version>4.1.0</version>
       </dependency>
+      <dependency>
+        <groupId>org.bdgenomics.adam</groupId>
+        <artifactId>adam-codegen_${scala.version.prefix}</artifactId>
+        <version>${project.version}</version>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 
   <profiles>
     <profile>
+      <id>python</id>
```

Comment:
Python is through PyPi and the python profile will not be active during a release (release scripts are unchanged).